controllers/auth_config_controller.go

```diff
@@ -441,142 +460,170 @@ func (r *AuthConfigReconciler) translateAuthConfig(ctx context.Context, authConf
 				SharedSecret: sharedSecret,
 				Permission:   *getJsonFromStaticDynamic(&authzed.Permission),
 			}
-			translatedAuthzed.Subject, translatedAuthzed.SubjectKind = authzedObjectToJsonValues(authzed.Subject)
-			translatedAuthzed.Resource, translatedAuthzed.ResourceKind = authzedObjectToJsonValues(authzed.Resource)
+			translatedAuthzed.Subject, translatedAuthzed.SubjectKind = spiceDBObjectToJsonValues(authzed.Subject)
+			translatedAuthzed.Resource, translatedAuthzed.ResourceKind = spiceDBObjectToJsonValues(authzed.Resource)
 
 			translatedAuthorization.Authzed = translatedAuthzed
 
-		case api.TypeUnknown:
+		case api.UnknownAuthorizationMethod:
 			return nil, fmt.Errorf("unknown authorization type %v", authorization)
 		}
 
 		interfacedAuthorizationConfigs = append(interfacedAuthorizationConfigs, translatedAuthorization)
+		authzIndex++
 	}
 
 	interfacedResponseConfigs := make([]auth.AuthConfigEvaluator, 0)
 
-	for _, response := range authConfig.Spec.Response {
-		translatedResponse := evaluators.NewResponseConfig(
-			response.Name,
-			response.Priority,
-			buildJSONExpression(authConfig, response.Conditions, jsonexp.All),
-			string(response.Wrapper),
-			response.WrapperKey,
-			response.Metrics,
-		)
+	if authConfig.Spec.Response != nil {
```

Comment:
nit: It may help avoid some of the repetition further below.

```suggestion
	if responseConfig := authConfig.Spec.Response; responseConfig != nil {
```